AISKU/rollup.config.js

```diff
@@ -168,12 +120,16 @@ updateDistEsmFiles(replaceValues, banner);
 export default [
   nodeUmdRollupConfigFactory(true),
   nodeUmdRollupConfigFactory(false),
-  browserRollupConfigFactory(true),
-  browserRollupConfigFactory(false),
+  browserRollupConfigFactory(true, majorVersion),
+  browserRollupConfigFactory(false, majorVersion),
   browserRollupConfigFactory(true, version),
   browserRollupConfigFactory(false, version),
-  browserCJSRollupConfigFactory(true),
-  browserCJSRollupConfigFactory(false),
-  browserCJSRollupConfigFactory(true, version),
-  browserCJSRollupConfigFactory(false, version)
+  browserRollupConfigFactory(true, majorVersion, 'cjs', '.cjs'),
+  browserRollupConfigFactory(false, majorVersion, 'cjs', '.cjs'),
+  browserRollupConfigFactory(true, version, 'cjs', '.cjs'),
+  browserRollupConfigFactory(false, version, 'cjs', '.cjs'),
+  browserRollupConfigFactory(true, majorVersion, 'iife', '.gbl'),
```

Comment:
Publishing the iife format as "gbl" to 
- sort of align with what VueJs is call (they use .global)
- As this explicitly creates the Microsoft namespace a global highlighting that this will add global scope variables.

Reply:
Do we still create cjs files?

Reply:
Yes (lines 128-130), as it would be useful for people that want to "install" it as a module via the CDN.